Formula/serve.rb

```diff
@@ -0,0 +1,15 @@
+class Serve < Formula
+  desc "Static http server anywhere you need one"
+  homepage "https://github.com/syntaqx/serve"
+  url "https://github.com/syntaqx/serve/releases/download/v0.2.0/serve_0.2.0_macos_x86_64.tar.gz"
```

Comment:
Formulae in the core tap should be built from source.

Reply:
👍